pvlib/iotools/tmy.py

```diff
@@ -160,7 +162,7 @@ def read_tmy3(filename, coerce_year=None, recolumn=True):
     head = ['USAF', 'Name', 'State', 'TZ', 'latitude', 'longitude', 'altitude']
 
     try:
-        with open(str(filename), 'r') as fbuf:
+        with open(str(filename), 'r', encoding=encoding) as fbuf:
             firstline, data = _parse_tmy3(fbuf)
```

Comment:
I'm partially of a mind to suggest a few useful encodings in the docstring, including the one we found to work for that SolarAnywhere file, and in the code leave it completely up to the user to pass in a suitable encoding.  So that would mean removing the `except` block below.  What do you think?

Reply:
I'm all for it - it simplifies things! Apologies for not listening the first time around 😄